cultivation/scripts/literature/metrics_literature.py

```
@@ -0,0 +1,166 @@
+        '--output',
+        type=str,
+        default=str(Path(__file__).parent.parent.parent / 'literature' / 'reading_stats.parquet'),
+        help='Output path for Parquet file'
+    )
+    args = parser.parse_args()
+
+    md_dir  = Path(args.metadata_dir)
+    db_path = Path(args.db_path)
+    out     = Path(args.output)
+    logging.info('Loading metadata and session data')
+    metadata_df = load_metadata(md_dir)
```

Ruff (0.11.9)

Local variable `metadata_df` is assigned to but never used

Remove assignment to unused variable `metadata_df`

(F841)